articles/python-projects-for-beginners/index.md

```diff
@@ -0,0 +1,196 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/python-projects-for-beginners
+title: Python Projects for Beginners
+description: This article provides a guide on building simple Python projects and working with basic concepts in Python. 
+author: anita-achu
+date: 2021-01-15T00:00:00-03:00
+topics: [Python]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/python-projects-for-beginners/hero.jpg
+    alt: python example image
+---
+
+In learning any programming language, the first step is learning the basic concepts of the language. While this is important, theoretical learning without working on real projects is not sufficient. The best way to have a good understanding of the language's basic concepts is by creating real-world projects with the learned concepts. This will help the programmer understand the language give a real insight into how the different tools are used and how they work. Working on projects makes learning a lot easier and helps you build your confidence, also choosing what to build is important. It is best to work with a project that involves using the concepts of the programming language.
+
+Like any programming language, the best practice for learning Python is building simple projects, which we will be doing in this tutorial.
+
+Remember that Python is a beginner-friendly language.
+
+## Introduction

```

Comment:
Also, we use headers H3 and beyond in all the articles. Therefore, update all the headers.